flow/scripts/variables.yaml

```diff
@@ -769,11 +769,15 @@ CTS_BUF_DISTANCE:
     Distance (in microns) between buffers.
   stages:
     - cts
+  tunable: 1
+  type: float
 CTS_BUF_LIST:
   description: |
     List of cells used to construct the clock tree. Overrides buffer inference.
   stages:
     - cts
+  tunable: 1
+  type: float
```

Comment:
@jeffng-or @maliberty This is confusing. From the description, I get a sense that this is a list of strings, but it is type float

Reply:
It will have to be in another PR as this one got merged.